rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -234,18 +265,112 @@ For example, if you work in Buenos Aires (ART (UTC -3)), specify a schedule that
++
+If you need a simpler interval, use *Interval*.
++
+You can use the Cron expression helper to compile the expresssion and check the natural language schedule summary.
++
+.What is a Cron expression?
+[%collapsible]
+=====
+
+'''
+
+Cron is a widely used standard for describing time and date information. You use a cron expression to define complex intervals, for example, `Run my process on minutes 3, 11, and 48 of every hour between 08:00 AM and 10:42 PM, only on Monday and Friday, only in February until April, and in November.
+
+The scheduler keeps track of every minute and starts the process run when the Cron expression matches the bot's time-date setting. You can trigger the event just once or at regular intervals.
+
+A date-time expression in the Cron format used by RPA Manager consists of five required settings in the following order:
+
+. Minutes (`0`-`59`)
+. Hours (`0`-`23`)
+. Day of month (`1`-`31`)
+. Month (`1`-`12`)
+. Day of the week (`0`-`7`), where Sunday is either `0` or `7`
+
+The Scheduler component also supports the following special characters:
+
+* `*`: All values within the range of valid values.
+* `-`: Range of values within the range of valid values, for example, `1-3`.
+* `,`: Additional values within the range of valid values, for example, `1,7`.
+* `/`: Incremental values within the range of valid values starting with the first specified value, for example, `1/7`.
+
+You can set either the day of the week or day of the month, but not both. Replace either the day of the week or day of the month with an asterisk.
+
+The Scheduler supports most standard Cron expressions. Here are a few good and bad examples:
+
+[%header,cols="3*"]
+|===
+|Expression |Behavior |Explanation 
+|`5 4 3 2 1` | At 04:05 AM on every Monday in every February and (?) on the 3rd of every February| This Cron expression is invalid. You can set either the day of the week or day of the month, but not both. Replace either the day of the week or day of the month with an asterisk. If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week. See next two examples. 
+|`5 4 3 2 *` | At 04:05 AM on the 3rd of every February| If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week. You can't specify the year. 
+|`5 4 * 2 3` | At 04:05 AM on every Wednesday of every February| If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week.You can't specify the year. 
+|`*/7 * * * *` |Run at minute 00, 07, 14, 21, 28, 35, 42, 49, 56 of every hour (of every day of every month on every weekday). |Starts at minute 0 because it's the first of all values in the valid range (*).
```

Comment:
```suggestion
|`*/7 * * * *` |At minute 00, 07, 14, 21, 28, 35, 42, 49, 56 of every hour (of every day of every month on every weekday). |Starts at minute 0 because it's the first of all values in the valid range (*).
```